src/Redux/Actions/verify.action.js

```diff
@@ -0,0 +1,26 @@
+import constants from '../constants';
+import apis from '../../Api';
+
+const VerifyUser = token => {
+	return async dispatch => {
+		dispatch({
+			type: constants.VERIFY_PENDING,
+			pending: true,
+		});
+		const result = await apis.verifyUser(token);
+		if (result.status === 200) {
+			return dispatch({
+				type: constants.VERIFY_SUCCESS,
+				verifyResult: result,
+			});
+		}
+		if (result.status === 401) {
+			return dispatch({
+				type: constants.VERIFY_ERROR,
+				error: result,
+			});
+		}
```

Comment:
what if there is another cases like: `connection-reset` or un-expected `500`. the page will be `pending` forever